src/github.com/couchbaselabs/sync_gateway/rest/api.go

```diff
@@ -142,6 +142,18 @@ func (h *handler) handleGetDB() error {
 	return nil
 }
 
+// Stub handler for hadling create DB on the public API returns HTTP status 412
+// if the db exists, and 403 if it doesn't.
+// fixes issue #562
+func (h *handler) handleCreateTarget() error {
+	dbname := h.PathVar("targetdb")
+	if _, err := h.server.GetDatabase(dbname); err != nil {
+		return base.HTTPErrorf(http.StatusForbidden, "does not exist")
+	} else {
+		return base.HTTPErrorf(http.StatusPreconditionFailed, "exists")
+	}
```

Comment:
For consistency I was following the style in admin_api.go for example:

handleCreateDB
or
handleDeleteDB